pages-e2e/vitest.config.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="vitest" />
```

Comment:
Why is this needed? There are no ambient types being used here.

Reply:
it's needed because it augments the UserConfig type:
![Screenshot 2024-06-26 at 10 20 49](https://github.com/cloudflare/next-on-pages/assets/61631103/a26e8171-8294-4fd0-9822-ae4982f1e5f1)

Without it typescript is not happy about the `test` field:
![Screenshot 2024-06-26 at 10 20 30](https://github.com/cloudflare/next-on-pages/assets/61631103/3a7f4171-6481-4c6e-a4c5-861c33c2d19a)

Reply:
Is that not solved by adding it to a tsconfig.json? Not blocking this, of course. Just interested.

Reply:
yeah I feel like that'd be an alternative solution, I don't feel like one is necessarily better than the other, I can give that a quick try 🙂 👍

Reply:
`tsconfig` updated instead 🙂👍

Reply:
actually just including this file in the `tsconfig` was enough to fix the issue 😅 

I guess typescript is picking up the vitest `d.ts` files by itself already 🤔